internal/migration/up/up.go

```diff
@@ -23,7 +23,18 @@ func Run(ctx context.Context, includeAll bool, config pgconn.Config, fsys afero.
 	if err != nil {
 		return err
 	}
-	return migration.ApplyMigrations(ctx, pending, conn, afero.NewIOFS(fsys))
+
+	err = migration.ApplyMigrations(ctx, pending, conn, afero.NewIOFS(fsys))
+	if err != nil {
+		return err
+	}
+
+	repeatableMigrations, err := migration.ListRepeatableMigrations(utils.MigrationsDir, afero.NewIOFS(fsys))
```

Comment:
I think it's worth moving this to inside GetPendingMigrations so other commands which depend on migrations can have these to. For eg. db push and db diff.

Reply:
Yep, will move it into `GetPendingMigrations`.

IMO, squashing would have to ignore repeatable migrations. Otherwise we are mixing versioned migrations (usually not repeatable) with repeatable migrations. Another option would be squashing repeatable migrations separately as in squash all versioned migrations together and squash all repeatable migrations together.